lib/MemoryMeasurement.cxx

```diff
@@ -0,0 +1,51 @@
+///\file Contains utilities to measure memory usage.
+#include <iostream>
+#include <cstdlib>
+#include <string>
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <unistd.h>
+#include <chrono>
+#include <stdlib.h>
+#include "rootbench/RBConfig.h"
+#include "rootbench/MemoryMeasurement.h"
+
+std::string getFileName(std::string path) {
+	std::string base_fn = path.substr(path.find_last_of("/\\") + 1);
+	return base_fn.substr(0, base_fn.find_last_of('.'));
+}
+
+std::string getMemoryStats(const std::string& dir, const std::string& filename, std::string& rootInvocation, const std::string& rootexe, const std::string& endInvoc) {
+		std::string perffile="";
+		if (access("/usr/bin/time", X_OK) == 0) {
```

Comment:
We should not hardcode this path `/usr/bin/time`, we should use a more standard way to resolve a system binary.

Reply:
alright! I will try to change it.